src/any_agent/evaluation/evaluate.py

```diff
@@ -10,7 +10,7 @@
 from any_agent.evaluation.results_saver import save_evaluation_results
 from any_agent.evaluation.test_case import TestCase
 from any_agent.logging import logger
-from any_agent.telemetry.telemetry import TelemetryProcessor
+from any_agent.telemetry.base import TelemetryProcessor
```

Comment:
How do you feel about editing the __init__ so that it can become 

```suggestion
from any_agent.telemetry import TelemetryProcessor
```